hkmc2DiffTests/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -162,11 +168,28 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
       
       // * Sometimes the JS block won't execute due to a syntax or runtime error so we always set this first
       host.execute(s"$resNme = undefined")
+
+      val timingId = runtimeClock.get.flatMap(s => if s == "" then N else S(s))
+      val startTime = timingId.map(_ => System.nanoTime())
       
       mkQuery(preStr, jsStr): stdout =>
```

Comment:
Is this the right thing to time?

Reply:
Can we just do the timing in node.js? Add a module with a `time` function that prints the elabsed time:

```
time of () =>
  foo(123)
> 456ms
= 42
```

Reply:
that would probably make more sense, and this could be frozen. at the moment, I'm not sure if stateful comments are particularly useful for anything other than freeze

Reply:
Yeah, I would suggest only using `freeze` instead.